client/components/tinymce/plugins/contact-form/dialog/settings.jsx

```diff
@@ -42,14 +42,11 @@ export default React.createClass( {
 				<SectionHeader label={ this.translate( 'Contact Form Notification Settings' ) } />
 				<Card>
 					<p>
-						{ this.translate(
-							'If you don’t make any changes here, feedback will be sent to the author of the %(postType)s and the subject will be the name of this %(postType)s.',
-							{
-								args: {
-									postType: this.props.postType === 'post' ? this.translate( 'post' ) : this.translate( 'page' )
-								}
-							}
-						) }
+					{
+						this.props.postType === 'post'
+						? this.translate( 'If you don’t make any changes here, feedback will be sent to the author of the post and the subject will be the name of this post.' )
+						: this.translate( 'If you don’t make any changes here, feedback will be sent to the author of the page and the subject will be the name of this page.' )
+					}
```

Comment:
Explanation: For example in German "page" ("Seite") is feminine and "post" masculine ("Beitrag"). 

- `this.translate( 'post' )` would be `Beitrag`
- `this.translate( 'page' )` would be `Seite`
- `this.translate( '...author of the post...' )` would be `Autor des Beitrags`
- `this.translate( '...author of the page...' )` would be `Autor der Seite`
- `this.translate( '...author of the %(postType)s' )` needs to be `Autor des %(postType)s` or  `Autor der %(postType)s` which is impossible